gordo_components/data_provider/iroc_reader.py

```diff
@@ -59,14 +59,12 @@ def load_series(
             map(lambda tag: IrocReader.base_path_from_asset(tag.asset), tag_list)
         )
         if len(set(base_paths_from_assets)) > 1:
-            logger.warning(
-                "Iroc reader found more than one asset from the tag list provided, returning none."
+            raise ValueError(
```

Comment:
Also log error, before raising the error

Reply:
Not sure I understand, I did the same as for the time check (line 53-56)!

Reply:
I see. Just thought that it was nice to do a log message first, and then raise. But now am not sure what is the best practice. Just leave it for now, no problem.